### PR TITLE
sqlitebrowser-nightly: Add 16-Jun-2022

### PR DESCRIPTION
**Pull request by rasa**
- [X] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rashil2000** [comment]
I think it will be better to use versioned URLs instead of the `latest` one, like https://nightlies.sqlitebrowser.org/win64/DB.Browser.for.SQLite-2022-06-16-win64.zip

**Comment by rasa** [comment]
> I think it will be better to use versioned URLs instead of the `latest` one, like https://nightlies.sqlitebrowser.org/win64/DB.Browser.for.SQLite-2022-06-16-win64.zip

Agreed. I will change the version to `2022.06.16` too.

File: bucket/sqlitebrowser-nightly.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        "32bit": {
+            "url": "https://nightlies.sqlitebrowser.org/win32/DB.Browser.for.SQLite-2022-06-18-win32.zip",
+            "hash": "edc4f1ded7b6cedacdd43a25cbd936ffc0ff4e2db54e4e49196fcc7609c2d0f0"
+        }
+    },
+    "extract_dir": "DB Browser for SQLite",
+    "bin": [
+        "DB Browser for SQLite.exe",
+        "DB Browser for SQLCipher.exe"
+    ],
```
Comment (rashil2000) [inline bucket/sqlitebrowser-nightly.json:20]: Do the binaries accept CLI args?

Comment (rasa) [reply bucket/sqlitebrowser-nightly.json:20]: They [do](https://github.com/sqlitebrowser/sqlitebrowser/wiki/Command-Line-Interface)!

**Pull request merged**